<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectorProperties</class>
 <widget class="QDockWidget" name="ConnectorProperties">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>262</width>
    <height>338</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>524287</width>
    <height>524287</height>
   </size>
  </property>
  <property name="allowedAreas">
   <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
  </property>
  <property name="windowTitle">
   <string>Connector Properties</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0,0,0,1">
    <property name="margin">
     <number>4</number>
    </property>
    <item>
     <widget class="QLabel" name="shapePaddingLabel">
      <property name="text">
       <string>Shape routing padding:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="shapePaddingSlider">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>23</height>
       </size>
      </property>
      <property name="maximum">
       <number>40</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="roundedCornersLabel">
      <property name="text">
       <string>Segment corner rounding:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="roundedCornersSlider">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>23</height>
       </size>
      </property>
      <property name="maximum">
       <number>30</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="nudgingCheckBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Nudge orthogonal routes</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>15</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Connector routing penalties</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="rightMargin">
        <number>12</number>
       </property>
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout" columnstretch="0,1">
         <property name="sizeConstraint">
          <enum>QLayout::SetMaximumSize</enum>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <item row="1" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Segment:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_2">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Crossing:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_3">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Path overlap</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSlider" name="segmentPenaltySlider">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>23</height>
            </size>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSlider" name="crossingPenaltySlider">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>23</height>
            </size>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QSlider" name="pathOverlapPenaltySlider">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>23</height>
            </size>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::MinimumExpanding</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>0</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
